libs/components/pages/src/lib/modules/link-list/link-list-item.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+
+/**
+ * A wrapper for each link in a link list.
+ * @internal
```

Comment:
The story mentions adding the link components to the "page" docs. Are you keeping this internal until everything's ready to go?

Reply:
The `@internal` isn't really needed. Removed.